DotNet/Report/Program.cs

```diff
@@ -234,31 +223,59 @@ static void RegisterServices(WebApplicationBuilder builder)
         c.DocumentFilter<HealthChecksFilter>();
     });
 
-    // Add kafka wrappers
-    //builder.Services.AddSingleton<IKafkaWrapper<Ignore, MeasureReportCreatedMessage, Null, Ignore>, KafkaWrapper<Ignore, MeasureReportCreatedMessage, Null, Ignore>>();
-    //builder.Services.AddSingleton<IKafkaWrapper<Ignore, ReportRequestedMessage, Null, ReportScheduledMessage>, KafkaWrapper<Ignore, ReportRequestedMessage, Null, ReportScheduledMessage>>();
-    builder.Services.AddTransient<IKafkaProducerFactory<string, AuditEventMessage>, KafkaProducerFactory<string, AuditEventMessage>>();
+    // Add Redis
+    var redisHost = builder.Configuration.GetConnectionString("Redis").Split(':')[0];
```

Comment:
Make sure this Redis configuration is mentioned in the `Report` service's documentation, or at least references the "Common Configuration" for Redis (assuming it follows those common configurations).